### PR TITLE
docs(aio): fix weird text

### PR DESCRIPTION
**Pull request by maxime1992**

### CODE REVIEW

File: aio/content/guide/reactive-forms.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ which does something with them (such as saving them)
 and returns a new _data model_ to the component that reflects the updated model state.
 
 Using reactive form directives does not require you to follow all reactive priniciples,
-but it does facilitate the reactive programming approach should you choose to use it.
+but it does facilitate the reactive programming approach.
```
Comment (gkalpak) [inline aio/content/guide/reactive-forms.md]: How is this text weird?

File: aio/content/guide/reactive-forms.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ which does something with them (such as saving them)
 and returns a new _data model_ to the component that reflects the updated model state.
 
 Using reactive form directives does not require you to follow all reactive priniciples,
```
Comment (gkalpak) [inline aio/content/guide/reactive-forms.md]: While you're at it:
_priniciples_ --> _principles_

File: aio/content/guide/reactive-forms.md (hunk 1 above)
Comment (maxime1992) [reply aio/content/guide/reactive-forms.md]: Well it seems that it's a way to speak in english that I'm not used to so maybe this PR doesn't make sense!

Comment (gkalpak) [reply aio/content/guide/reactive-forms.md]: Maybe adding a comma before `should` would make it more easy to read (but I am not a native speaker, so I am not sure if it syntactically correct :grin:).

Comment (petebacondarwin) [reply aio/content/guide/reactive-forms.md]: I think a comma is adequate here; and a fix to the `priniciples` typo above.

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

File: aio/content/guide/reactive-forms.md (hunk 1 above)
Comment (kapunahelewong) [reply aio/content/guide/reactive-forms.md]: I agree with @petebacondarwin on both points. The phrase "...should you choose to use it" is standard English, though it is an advanced construction, which might be difficult for non-native speakers.

For the grammatically curious, this is a construction using inversion and an omission of the conditional "if". "Should" is an example of the subjunctive mood (conditional tense of "shall"). For more info, checkout these Wikipedia entries:

[English Conditional Sentences](https://en.wikipedia.org/wiki/English_conditional_sentences#Inversion_in_condition_clauses)

[Inversion and the Subjunctive](https://en.wikipedia.org/wiki/English_subjunctive#Inversion)

So, I like the comma, which allows the "should" phrase to stay. However, we could clarify it for non-native speakers with something like this:

> ...the reactive programming approach, if you were to use it.

Thoughts?

**Comment by brandonroberts** [comment]
Closing as outdated since the reactive forms guide has been rewritten.

**Pull request closed without merging**